sass/theme.scss

```diff
@@ -190,6 +194,14 @@ select {
     width: auto;
   }
 
+  .add {
```

Comment:
No need to change in this breakpoint anything. Add button should be the same all the time.